api/agent/agent.go

```diff
@@ -218,6 +218,7 @@ func NewDockerDriver(cfg *Config) (drivers.Driver, error) {
 		PreForkUseOnce:       cfg.PreForkUseOnce,
 		PreForkNetworks:      cfg.PreForkNetworks,
 		MaxTmpFsInodes:       cfg.MaxTmpFsInodes,
+		MaxImageCacheSize:    cfg.MaxImageCacheSize,
```

Comment:
If you're still on board with switching to an approach that looks at the amount of free space on a volume, would it be worth it to change this to a percentage of space free/in-use instead of using an absolute size?  This would allow you to compute the amount of free space needed without needing to rely on knowing the absolute size of the docker volume as a config parameter.